### ISSUE TITLE
AbstractionLocalizer -> LocalizerAbstraction

### ISSUE THREAD

**Issue by ferponcem** [individual-brain-charting/public_analysis_code#63]
Modifying the identifier from AbstractionLocalizer to LocalizerAbstraction to match the filenames:

In the cluster files are as follows: `/derivatives/sub-04/ses-45/res_task-LocalizerAbstraction_*`

Inside **ibc_data** I'm changing the string in `ibc_tasks`, `ibc_conditions`, `all_contrasts`, `main_contrast` and `contrast/AbstractionLocalizer`, to match the data.

**Comment by ferponcem** [comment]
Reopening because I forgot to mention we have not added any of the contrast from **LocalizerAbstraction** to `main_contrasts`. Should we?
See [contrast list](https://github.com/individual-brain-charting/public_analysis_code/blob/master/ibc_data/all_contrasts.tsv#L660) for this task.

**Comment by bthirion** [comment]
Indeed. Can you add the *-other + reponse contrasts to `main_contrasts` ?
That would help, thx.
